### PR TITLE
feat: add the Stryker logo to the badge

### PR DESCRIPTION
**Pull request by regseb**
Now that the Stryker logo is in [Simple Icons](https://github.com/simple-icons/simple-icons/pull/7388), I suggest adding it to the dashboard badge: Mutation testing badge

I've colored it white (with the `whitesmoke`/`#F5F5F5` value used by [Shields](https://github.com/badges/shields/blob/ae018cd/lib/load-simple-icons.js#L21)) because the default `#E84B39` orange isn't [accessible](https://webaim.org/resources/contrastchecker/?fcolor=E84B39&bcolor=555555).

### CODE REVIEW

**Comment by rouke-broersma** [comment]
That's cool, thank you! @nicojs what do you think

**Comment by mthmulders** [comment]
The badge can also be generated from coverage reports that aren't generated by Stryker (in fact, I'm maintaining a reporter for Pitest that does just that).

Would it be a bit odd to see the Stryker logo in such cases?

**Comment by rouke-broersma** [comment]
> The badge can also be generated from coverage reports that aren't generated by Stryker (in fact, I'm maintaining a reporter for Pitest that does just that).
> 
> Would it be a bit odd to see the Stryker logo in such cases?

Well it's a badge hosted by the Stryker dashboard so imo it's not that strange. But we could make this customizable or toggleable.

**Comment by mthmulders** [comment]
> > The badge can also be generated from coverage reports that aren't generated by Stryker (in fact, I'm maintaining a reporter for Pitest that does just that).
> > Would it be a bit odd to see the Stryker logo in such cases?
> 
> Well it's a badge hosted by the Stryker dashboard so imo it's not that strange. But we could make this customizable or toggleable.

Agree, it doesn't fall out of thin air. I'm just double-checking whether it's something to consider or not. I'll leave it up to you to decide upon it.

**Comment by hugo-vrijswijk** [comment]
Should be noted that you can always change the logo from the query string `namedLogo`, if someone really doesn't want it

File: packages/badge-api/badge/ShieldMapper.ts [hunk 1]
```diff
@@ -18,17 +18,21 @@ export class ShieldMapper {
...
     } else {
       return {
-        color: Color.Grey,
+        schemaVersion: 1,
         label: 'Mutation score',
         message: 'unknown',
-        schemaVersion: 1,
+        color: Color.Grey,
+        namedLogo: 'stryker',
+        logoColor: 'whitesmoke',
```
[the remark this answers is not in the document]
Comment (regseb) [reply packages/badge-api/badge/ShieldMapper.ts]: Done. https://github.com/stryker-mutator/stryker-dashboard/pull/384/commits/e4da0574415a43b4eeb7d0bb89b16e7da78aaffd

**Pull request merged**